src/lib/degradeableItems.ts

```diff
@@ -302,51 +302,68 @@ export async function degradeItem({
 	const newCharges = Math.floor(currentCharges - chargesToDegrade);
 
 	if (newCharges <= 0) {
-		// If no more charges left, break and refund the item.
+		// If the user runs out for charges for a degradeable item, break and refund the item.
 		const hasEquipped = user.gear[degItem.setup].hasEquipped(item.id, false);
-		const hasInBank = user.owns(item.id);
+		const hasInBank = user.hasEquippedOrInBank(item.id);
```

Comment:
owns is preferred due to exclusively checking the bank (unless equipment variable is passed)

In this scenario owning a single charged scythe (equipped) would make it true for "hasEquipped" and "hasInBank". IF statements later filter this variable out due to equipped taking priority.

Reply:
This is .hasEquippedOrInBank since its the only bank function I could find that checks for similar items. There isn't a scenario where this wouldn't work that I can think of. The user has to have the equipable chargeditems equipped for it to lose charges.